prime-dotnet-webapi/Models/DoNotEmail.cs

```diff
@@ -0,0 +1,16 @@
+using System;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace Prime.Models
+{
+    [Table("DoNotEmail")]
+    public class DoNotEmail : BaseAuditable
+    {
+        [Key]
+        public int Id { get; set; }
+        public string Email { get; set; }
+        public DateTimeOffset ModifiedDate { get; set; }
```

Comment:
Since the final table structure already include audit fields (including UpdatedTimeStamp), I think ModifiedDate is not needed.